### DISCUSSION TITLE
zookeeper keep crashing

### DISCUSSION THREAD

**Discussion by ranyhb** [strimzi/strimzi-kafka-operator#8296] [General]
i have a bug with my statefulset of the zookeeper.
one of the pods always crash for the reason that the current epoch is older than the last zxid.
the classic way to solve that in the zookeeper is to move the data folder to a backup folder then the zookeeper will pull the data from another running zookeeper. In our case, using the strimzi, i cant do that because the pod crash every time i try to enter the command.
i tried also creating a pod using the same PV in order to do that but i could because of permissions.
do you have anymore ideas how can i solve that?

**Comment by scholzj** [comment]
You probably need to share more details. Full configuration (Kafka CR), full logs etc.
